service/controller/v11/key/key.go

```diff
@@ -399,11 +399,11 @@ func ImageID(customObject v1alpha1.AWSConfig) (string, error) {
 		Current Release: CoreOS Container Linux stable 1576.5.0 (HVM)
```

Comment:
It would be nice to update the version in the comment.

Reply:
Good catch. I'll fix.

Reply:
Done.